src/Compilers/CSharp/Test/Syntax/Parsing/MemberDeclarationParsingTests.cs

```diff
@@ -34,6 +35,10 @@ private SyntaxTree UsingTree(string text, CSharpParseOptions options, params Dia
             return tree;
         }
 
+        private static readonly CSharpParseOptions RequiredMembersOptions = TestOptions.RegularPreview;
+        public static readonly IEnumerable<object[]> ParseOptionsDataWithPreview = new[] { new[] { TestOptions.Regular }, new[] { RequiredMembersOptions }, new[] { TestOptions.Script } };
+        public static readonly IEnumerable<object[]> ParseOptionsData = new[] { new[] { TestOptions.Regular }, new[] { TestOptions.Script } };
```

Comment:
It might be easier to understand these if they literally referred to what set of options are tested. For example: `... IEnumerable<object[]> RegularAndScript = new[] { new[] { TestOptions.Regular }, new[] { TestOptions.Script } }`.

Also, I think we now use `TestOptions.RegularNext` when we want "preview while feature is under development, then minimum supported version once feature is shipped".